dashboard/src/modules/components/FooterComponent/index.jsx

```diff
@@ -0,0 +1,19 @@
+import "./index.less";
+
+import React from "react";
+import { useSelector } from "react-redux";
+
+const Footer = () => {
+  const { endpoints } = useSelector((state) => state.apiEndpoint);
+  const versionNumber = endpoints?.identification
+    ?.split(" ")
```

Comment:
So `"Pbench server 1.0.0-a2d0f0196"` becomes `["Pbench", "server", "1.0.0-a2d0f0196"]` becomes `["1.0.0-a2d0f0196"]` becomes `"1.0.0-a2d0f0196"`.

The `split`/`slice`/`join` is interesting although it feels a bit obscure: especially using `join` essentially just to extract the single element of an array. 😆 

Another option might be `split(" ").pop()` which should directly return `"1.0.0-a2d0f0196"`

Reply:
That's right. Too much of confusion there :grinning: